src/routes/delegates/delegates.controller.spec.ts

```diff
@@ -0,0 +1,151 @@
+      imports: [
+        // feature
+        DelegatesModule,
+        // common
+        DomainModule,
+        TestCacheModule,
+        TestConfigurationModule,
+        TestNetworkModule,
+      ],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    app.useGlobalFilters(new DataSourceErrorFilter());
+
+    await app.init();
+  });
+
+  describe('GET delegates for a Safe', () => {
+    it('Success', async () => {
+      const safe = '0x5afe3855358e112b5647b952709e6165e1c1eeee';
+      const chainId = '1';
+      const chainResponse = chainFactory(chainId);
+      mockNetworkService.get.mockResolvedValueOnce({ data: chainResponse });
+      const pageDelegates = <Page<Delegate>>{
+        count: 2,
+        results: [delegateFactory(safe), delegateFactory(safe)],
+      };
+      mockNetworkService.get.mockResolvedValueOnce({ data: pageDelegates });
+
+      await request(app.getHttpServer())
+        .get(`/chains/${chainId}/delegates?safe=${safe}`)
+        .expect(200)
+        .expect(pageDelegates);
+
+      expect(mockNetworkService.get).toBeCalledTimes(2);
+      expect(mockNetworkService.get.mock.calls[0][0]).toBe(
+        'https://test.safe.config/api/v1/chains/1',
+      );
+      expect(mockNetworkService.get.mock.calls[1][0]).toBe(
+        `${chainResponse.transactionService}/api/v1/delegates/`,
+      );
+      expect(mockNetworkService.get.mock.calls[1][1]).toStrictEqual({
+        params: {
+          safe: safe,
+          delegate: undefined,
+          delegator: undefined,
+          label: undefined,
+          limit: undefined,
+          offset: undefined,
+        },
+      });
+    });
+
+    it('Should return empty result', async () => {
+      const safe = '0x5afe3855358e112b5647b952709e6165e1c1eeee';
+      const chainId = '1';
+      const chainResponse = chainFactory(chainId);
+      mockNetworkService.get.mockResolvedValueOnce({ data: chainResponse });
+      const pageDelegates = <Page<Delegate>>{
+        count: 0,
+        results: [],
+      };
+      mockNetworkService.get.mockResolvedValueOnce({ data: pageDelegates });
+
+      await request(app.getHttpServer())
+        .get(`/chains/${chainId}/delegates?safe=${safe}`)
+        .expect(200)
+        .expect(pageDelegates);
+
+      expect(mockNetworkService.get).toBeCalledTimes(2);
+      expect(mockNetworkService.get.mock.calls[0][0]).toBe(
+        'https://test.safe.config/api/v1/chains/1',
+      );
+      expect(mockNetworkService.get.mock.calls[1][0]).toBe(
+        `${chainResponse.transactionService}/api/v1/delegates/`,
+      );
+      expect(mockNetworkService.get.mock.calls[1][1]).toStrictEqual({
+        params: {
+          safe: safe,
+          delegate: undefined,
+          delegator: undefined,
+          label: undefined,
+          limit: undefined,
+          offset: undefined,
+        },
+      });
+    });
+
+    it('Should failure with bad request', async () => {
```

Comment:
nitpick: "Should fail"

Reply:
@fmrsabino should return a bad request, and returns 503 something is wrong

Reply:
> @fmrsabino should return a bad request, and returns 503 something is wrong

Your comment is unrelated with my comment right? Maybe I misunderstood. But are you saying that it should return 400 instead of 503 in this test?